ports/py-add-vcpkg-dll-path/sitecustomize.py

```diff
@@ -0,0 +1,8 @@
+import os
+import sys
+from pathlib import Path
+
+vcpkg_bin_path = Path(sys.prefix) / '/../../bin'
+if not vcpkg_bin_path.is_dir():
+  raise RuntimeError(f'Could not add "{vcpkg_bin_path}" to dll paths because it does not exist.')

```

Comment:
Does the error contain a call stack for the user? Maybe add a note: "If you copied the files from vcpkg either delete this file or adjust it accordingly"

Reply:
It gives a hint:

```
Error in sitecustomize; set PYTHONVERBOSE for traceback:
RuntimeError: Could not add "D:\..\..\bin" to dll paths because it does not exist.
```

https://github.com/open-vcpkg/python-registry/actions/runs/8264216708/job/22607351891#step:7:339

but I don't yet understand why it fails in this context

Reply:
```suggestion
  raise RuntimeError(f'Could not add "{vcpkg_bin_path}" to dll paths because it does not exist. If you copied the files from vcpkg either delete this file or adjust it accordingly.')
```